apps/judicial-system/web/src/routes/Court/Indictments/Completed/Completed.tsx

```diff
@@ -57,8 +64,19 @@ const Completed: FC = () => {
       return
     }
 
+    createEventLog({
```

Comment:
Is this event perhaps import enough that it is worth waiting for its successful completion before disploying the modal? An error would result in a toaster being shown and no modal?

Please consider if this change is appropriate.

Reply:
Yes you're right, I added this handling in 🙏